lib/internal/per_context/primordials.js

```diff
@@ -265,5 +281,5 @@ primordials.SafeStringIterator = createSafeIterator(
   primordials.StringIteratorPrototypeNext
 );
 
-Object.setPrototypeOf(primordials, null);
-Object.freeze(primordials);
+ObjectSetPrototypeOf(primordials, null);
```

Comment:
```suggestion
ReflectSetPrototypeOf(primordials, null);
```

Reply:
`Object.setPrototypeOf` throws a `TypeError` when changing the prototype was unsuccessful, whereas `Reflect.setPrototypeOf` only returns `false`.